scripts/example.coffee

```diff
@@ -114,74 +174,71 @@ module.exports = (robot) ->
   robot.respond /wake me up when september ends/i, (res) ->
     res.reply 'Deployyyyyyy....... to production :D'
 
-  robot.respond /ankit/i, (res) ->
-    checkouts = robot.brain.get('features')
-    result = ':realtor: CORE WEB FEATURE BOXES :realtor: \n\n' ;
-    for own feature, name of checkouts
-      result = result.concat("> *#{name}* checked out #{feature}\n");
-    result = result.concat("\n\n> Any *feature* not listed is free for the taking! :parrotcop: ")
-    res.send result 
+  robot.respond /migrate features/, (res) ->
+    res.send "Migrating to the new DB format (key `featuresv2`)..."
+    (new Checkouts(robot)).migrate(res)
+    res.send "Completed migration :tayne:"
 
   robot.respond /who checked out (.*)\??/i, (res) ->
     feature = res.match[1]
-    checkouts = robot.brain.get("features") || {}
-    name = checkouts[feature] || 'nobody'
-
-    res.send "#{name} checked out #{feature}"
+    res.send "#{new Checkouts(robot).who(feature)} checked out #{feature}"
 
   robot.respond /show checkouts/i, (res) ->
```

Comment:
Could wrap it in an if-statement to see if there are any records in `featuresv2` and if there are, use that, if not, use the old logic? I'm fine either way, though. wdyt?